assertionengine/assertion_engine.py

```diff
@@ -129,6 +129,10 @@ def verify_assertion(
     message: str = "",
     custom_message: Optional[str] = None,
 ) -> Any:
+    if operator is None and expected:
+        raise ValueError(
+            f"It is not possible perform assertion when operator is {operator} type({type_converter(operator)})"
+        )
```

Comment:
Good fix but could be enhanced in few different ways.

The biggest problem is that the error message isn't too clear. At least for me it's hard to understand even though I know the root cause. Additionally, it's also strange to use placeholders like `{operator}` when you know exactly what the value is. Some ideas for better message below:

> Invalid validation parameters. Assertion operator is mandatory when specifying expected value.

> Cannot perform validation. Giving only expected value, not assertion operator, is invalid.

It think it would also be a bit better to use `and expected is not None` instead of just `expected`. Using e.g. an empty string as the expected value ought to cause this same error.

Finally, the code could be structured like
```python
if operator is None:
    if expected is not None:
        raise ValueError(...)
    return value
```